tc-benchmark-tests.sh

```diff
@@ -23,6 +23,14 @@ exec_benchmark()
         --lm_binary /tmp/lm.binary \
         --trie /tmp/trie \
         --csv ${csv} \
+        --keep
+
+    python ${DS_ROOT_TASK}/DeepSpeech/ds/bin/benchmark_nc.py \
+        --dir /tmp/bench-ds/ \
+        --models ${model_file} \
+        --wav /tmp/LDC93S1.wav \
+        --alphabet /tmp/alphabet.txt \
+        --csv ${csv}
```

Comment:
We should output to a different CSV file :)

Reply:
I think it does relate to the failure there: https://tools.taskcluster.net/groups/PYJOg0bYRhSHgGhu7NAHpA/tasks/e2dRVYd5QAGqgi6HrUxuRA/runs/0